lerobot/common/policies/diffusion/configuration_diffusion.py

```diff
@@ -134,7 +134,7 @@ class DiffusionConfig:
     down_dims: tuple[int, ...] = (512, 1024, 2048)
     kernel_size: int = 5
     n_groups: int = 8
-    diffusion_step_embed_dim: int = 128
+    diffusion_step_embed_dim: int = 256
```

Comment:
I think this change should be reverted right? To keep in line with the default PushT policy.